### PR TITLE
split up skopeo man pages

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: docs/skopeo-standalone-sign.1.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## EXAMPLES
+
+```sh
+$ skopeo standalone-sign busybox-manifest.json registry.example.com/example/busybox 1D8230F6CDB6A06716E414C1DB72F2188BB46CC8 --output busybox.signature
+$
+```
+
+## SEE ALSO
+skopeo(1)
```
Comment (TomSweeneyRedHat) [inline docs/skopeo-standalone-sign.1.md]: probably should add skopeo-copy(1) too

File: docs/skopeo-inspect.1.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Return low-level information about _image-name_ in a registry
+
+  **--raw** output raw manifest, default is to format in JSON
+
+  _image-name_ name of image to retrieve information about
+
+  **--authfile** _path_
+
+  Path of the authentication file. Default is ${XDG_RUNTIME\_DIR}/containers/auth.json, which is set using `podmanlogin`.
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply docs/skopeo-inspect.1.md]: Fixed

File: docs/skopeo-manifest-digest.1.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+% skopeo-manifest-digest(1)
+
+## NAME
+skopeo\-manifest\-digest - Return low-level information about _image-name_ in a registry
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply docs/skopeo-manifest-digest.1.md]: Fixed

File: docs/skopeo-standalone-sign.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/skopeo-standalone-sign.1.md]: added

**Comment by rhatdan** [comment]
@TomSweeneyRedHat This is what you do when you are on a 7 hour flight...

**Comment by TomSweeneyRedHat** [comment]
No, when I'm on a 7 hour flight I'm praying to make it onto the ground in one piece.

**Comment by TomSweeneyRedHat** [comment]
Couple nits since your last edit.  I like the change, probably well overdue.  Nice work @rhatdan.

**Comment by rhatdan** [comment]
@runcom @vrothberg @mtrmac PTAL

**Comment by vrothberg** [comment]
@rhatdan can you update the branch?

**Comment by vrothberg** [comment]
It looks like something went wrong in your branch as the pushed commit is a merge commit.

**Comment by vrothberg** [comment]
Changes look good to me. To be honest, I never noticed that there's only a single manpage. Thanks a lot, @rhatdan!

**Pull request merged**